src/utils/tracked-controls.js

```diff
@@ -2,6 +2,17 @@ var DEFAULT_HANDEDNESS = require('../constants').DEFAULT_HANDEDNESS;
 var AXIS_LABELS = ['x', 'y', 'z', 'w'];
 var NUM_HANDS = 2; // Number of hands in a pair. Should always be 2.
 
+var splitOnPipeMap = {};
+
+function splitOnPipeOnce (stringToSplit) {
+  var rtn = splitOnPipeMap[stringToSplit];
+  if (!rtn) {
```

Comment:
Early return `if (rtn) { return; }`

Reply:
(I'm not sure it actually looks better now, but I understand your stylistic preference.)